modules/localgov_directories_location/config/override/views.view.localgov_directory_channel.yml

```diff
@@ -349,15 +345,14 @@ display:
             use_highlighting: false
             multi_type: separator
             multi_separator: ', '
-        location:
-          id: location
-          table: search_api_entity_geo_entity
-          field: location
-          relationship: localgov_location
+        search_api_rendered_item:
```

Comment:
I don't see any use of this `Search: Rendered item` field anywhere in this View's display.  If we drop this field, the `embed_map` display loads about a thousand map pins without any memory exhaustion issues in our site.  So I would request that this `search_api_rendered_item` field be dropped.

I am still running into memory exhaustion issues around 1500 map pins after dropping the `search_api_rendered_item` field.  But I don't yet see any easy way out of that.

Reply:
@ekes fancy adding this change?

Reply:
I'm almost tempted to go the other way. If we're not including a freetext search based on rendered item in the default view maybe we should be?